dev-utils/build.js

```diff
@@ -152,7 +152,10 @@ function getBabelConfig(bundleType, packageType) {
   let options = {
     comments: false,
     babelrc: false,
-    exclude: '/**/node_modules/**',
+    // We must exclude `parse5` from the packages we ignore
```

Comment:
nit: 

What do you think about adding the exclusion rule in a variable? Then you could the comment only once, something like:


```
   // We must exclude `parse5` from the packages we ignore
   // because otherwise it would not be transpiled. Therefore,
   // it would break IE11.
   const babelExclusionRule = /node_modules\/(?!parse5).+/,
```

Reply:
Great idea!